### ISSUE TITLE
rustup is not available after 'brew install rustup-init' on Appls Silicon

### ISSUE THREAD

**Issue by kuncevic** [rust-lang/rustup#3458]
### Problem

Description:
-

When installing rustup via Homebrew on an Apple Silicon Mac, the main rustup binary is not being installed in /opt/homebrew/bin.

Expected behavior:
-

The rustup installation should place the main rustup binary in `/opt/homebrew/bin`.

### Steps

Steps to reproduce:
-

1.   Install Rust via Homebrew

`brew install rust`

2.   Install rustup

`brew install rustup-init`

3.  Check for rustup binary

`ls /opt/homebrew/bin`

The rustup binary is not present, only the rustup-init executable.

### Rustup version

```console
rustup-init version: 

1.26.0
```

### Installed toolchains

```console
Environment:
-
    OS: macOS Ventura 13..4.1 (Apple Silicon)
    Install method: Homebrew
```

**Comment by rami3l** [comment]
@kuncevic Hi! You are on the right path so far! Please try to run `rustup-init` once to set everything up.

TLDR (originally from https://github.com/rust-lang/rustup/issues/3426#issuecomment-1683866214):

> My personal recommendation for setting up a `rust` environment on macOS would be:
> 1. Install `rustup-init` with `brew install rustup-init`;
> 2. Proceed with that `rustup-init` installation;
> 3. If you also wish to have `brew`-installed `rust`, make sure your `rustup`-installed `rust` comes first in the `PATH`, and optionally [configure the system toolchain](https://rust-lang.github.io/rustup/installation/package-managers.html) via `rustup toolchain link system <path>`.

Here's what a correctly-configured `brew`-managed `rustup` installation should look like:

```sh
> ls -lah $(which rustup)
... ~/.cargo/bin/rustup -> /opt/homebrew/bin/rustup-init
```

You can see that after running `rustup-init` and putting `~/.cargo/bin` in your `$PATH`, you have a `rustup` link that **points to** your homebrew `rustup-init` installation.

In fact, `rustup-init` **is** `rustup`, but with a different file name: https://github.com/rust-lang/rustup/blob/a4dd7d03501def7ee3e4e2961bca7f6ec3b2e1aa/src/bin/rustup-init.rs#L1-L12

**Comment by kuncevic** [comment]
Right I c, I believe there is a bit of gap in documentation.  Initially I've read desc onhttps://formulae.brew.sh/formula/rustup-init saying that `Also known as: rustup` however that was confusing...

All fixed through by `brew uninstall rust`, `brew unlink rust`, `brew install rustup-init` and then, as I am using fish shell, this https://github.com/rust-lang/rustup/issues/478#issuecomment-1288194464. 

Thank you for following up @rami3l

**Comment by rami3l** [comment]
@kuncevic Glad to hear that! Please feel free to follow up doc issues in #3460 :)

BTW could you please close this issue as resolved? Sadly I don't currently have the permission to do so. Many thanks!
